config/environments/production.rb

```diff
@@ -90,4 +90,14 @@
       :sender_address => %{"OpenStax Exchange" <noreply@openstax.org>},
       :exception_recipients => %w{exchange-dev@openstax.org}
     }
+
+  config.lograge.enabled = true
+  config.log_tags = [ :remote_ip ]
+  config.lograge.custom_options = lambda do |event|
+    params = event.payload[:params].reject do |k|
+      ['controller', 'action', 'format'].include? k
+    end
+    { "params" => params }
+  end
+  config.lograge.ignore_actions = ["static_pages#status"]
```

Comment:
In tutor-server, the `/status` endpoint is handled in the `StaticPagesController`, which is why this transferred config has `static_pages#status`.  But since the action is handled directly in the router, I don't know what the corresponding action to ignore would be. In your production-like setup, could put a breakpoint in the lograge gem to see what the action looks like.